### ISSUE TITLE
[record improvement] It would be better if we limit the length of the "--description" value while submitting the record

### ISSUE THREAD

**Issue by shilei-wang** [irisnet/irishub#452]
Improvement advice FYI

**Comment by yunkunrao** [comment]
Yes, because this field is one piece of on chain data. And storage resource is precious and limited, so I will add a length check for this field before submitting the record.

**Comment by yunkunrao** [comment]
Offload input params check to ValidateBasic()
PR[https://github.com/irisnet/irishub/pull/456]
